sdcm/cluster_docker.py

```diff
@@ -387,15 +410,51 @@ def __init__(self,
                                params=params)
 
     def node_setup(self, node: DockerNode, verbose=False, **kwargs):
-        node.wait_ssh_up(verbose=verbose)
         node.remoter.sudo("apt update", verbose=True, ignore_status=True)
         node.remoter.sudo("apt install -y openjdk-8-jre", verbose=True, ignore_status=True)
         node.remoter.sudo("ln -sf /usr/lib/jvm/java-1.8.0-openjdk-amd64/jre/bin/java* /etc/alternatives/java",
                           verbose=True, ignore_status=True)
 
+        self.install_sudo(node, verbose=verbose)
+        self._install_docker_cli(node, verbose=verbose)
         if self.params.get('client_encrypt'):
             node.config_client_encrypt()
 
+    def _install_docker_cli(self, node, verbose=False):
```

Comment:
this is for loader only, we shouldn't use that for scylla instances

Reply:
@dimakr what about this ? I can accept sudo is needed for something, but we need to remove this part

we shouldn't be installing docker cli into any docker instances